canonical-web-and-design/charmed-osm.com
# Copy update - charmed-osm.com

## Done

- Copy changes on - charmed-osm.com
- Add blog widget to home page

## QA

- Check out this feature branch
- Run the site using the command `./run`
- View the site locally in your web browser at: http://0.0.0.0:8042/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page against [copy doc](https://docs.google.com/document/d/1mKfttykmrc4ub_LkqdRGIE_x9WgR1NGr7ulxrLvIZXk/edit?ts=60224421#heading=h.mbee56yofefb)


## Issue / Card

Fixes [#9224](https://github.com/canonical-web-and-design/ubuntu.com/issues/9224)

## Screenshots

[if relevant, include a screenshot]
